app/presenters/idv/how_to_verify_presenter.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+class Idv::HowToVerifyPresenter
+  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::TranslationHelper
+
+  attr_reader :idv_session
+
+  def initialize(selfie_check_required:)
+    @selfie_required = selfie_check_required
+  end
+
+  def how_to_verify_info
+    if @selfie_required
```

Comment:
If we were to swap the unused `idv_session` `attr_reader` with an `attr_reader` for `selfie_required`, we could simplify these a bit to reference the getter method instead of the instance property.

```suggestion
    if selfie_required
```

Reply:
Resolved with commit 11ab0eb94b37b93e6dac68fb5898b63b74ba795c